src/Symfony/Component/Intl/README.md

```diff
@@ -15,11 +15,11 @@ The documentation for the component can be found [online] [1].
 Resources
 ---------
 
-You can run the unit tests with the following command:
-
-    $ cd path/to/Symfony/Component/Intl/
-    $ composer install
-    $ phpunit
+  * [Documentation](https://symfony.com/doc/current/components/intl.html)
+  * [Contributing](https://symfony.com/doc/current/contributing/index.html)
+  * [Report issues](https://github.com/symfony/symfony/issues) and
+    [send Pull Requests](https://github.com/symfony/symfony/pulls)
+    in the [main Symfony repository](https://github.com/symfony/symfony)
 
 [0]: http://www.php.net/manual/en/intl.setup.php
 [1]: https://symfony.com/doc/2.3/components/intl.html
```

Comment:
This documentation link should be removed in favor of the new one (especially given that it does not link to the current doc)